src/components/AddFood.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const AddFood = (props) => {
+    return (
+        <form id="addFoodForm" onSubmit={props.onAdd}>
```

Comment:
calling the function and working around the target for the inputs works perfectly fine, but when working with forms in react is common to work around the state, and changing the inputs dynamically. Similar to how you did for the quantity in the FoodBox. then just invoke the function passing the information in the state.